### PR TITLE
Breakpoints stage 7

### PR DESCRIPTION
**Pull request by kof**
## Description

1. resize handles on canvas using css only (visibly faster)
2. initial breakpoint on first load when base breakpoint exists needs to be selected in a way that the base breakpoint is activated, currently we were landing on a smaller breakpoint when there was not enough space to land on base
3. fix overflow for when instance on canvas is bigger than iframe size, outline is rendered outside of iframe and may run outside of it unless we hide the overflowing parts

## Code Review

- [ ] hi @istarkov , I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

File: apps/builder/app/builder/features/workspace/canvas-tools/canvas-tools.tsx [hunk 1]
```diff
@@ -46,7 +50,7 @@ export const CanvasTools = ({ publish }: CanvasToolsProps) => {
     const { dropTarget, placementIndicator } = dragAndDropState;
     const dropTargetInstance = instances.get(dropTarget.itemSelector[0]);
     return dropTargetInstance ? (
-      <Box css={toolsStyle}>
+      <div className={containerStyle({ overflow: "hidden" })}>
```
Comment (istarkov) [inline apps/builder/app/builder/features/workspace/canvas-tools/canvas-tools.tsx:53]: variant with same name as css looks weird

File: apps/builder/app/builder/features/workspace/canvas-tools/canvas-tools.tsx [hunk 2]
```diff
@@ -56,17 +60,19 @@ export const CanvasTools = ({ publish }: CanvasToolsProps) => {
...
   }
 
   return (
-    <Box css={toolsStyle}>
+    <div className={containerStyle()}>
       <MediaBadge />
       <ResizeHandles />
-      {isPreviewMode === false && <SelectedInstanceOutline />}
-      {isPreviewMode === false && <HoveredInstanceOutline />}
+      <div className={containerStyle({ overflow: "hidden" })}>
```
Comment (istarkov) [inline apps/builder/app/builder/features/workspace/canvas-tools/canvas-tools.tsx:71]: why overflow hidden as before it wasn't rendered at all?

**Review by istarkov — APPROVED** [review]

File: apps/builder/app/builder/features/workspace/canvas-tools/canvas-tools.tsx (hunk 1 above)
Comment (kof) [reply apps/builder/app/builder/features/workspace/canvas-tools/canvas-tools.tsx:53]: yeah, though its super clear

File: apps/builder/app/builder/features/workspace/canvas-tools/canvas-tools.tsx (hunk 2 above)
Comment (kof) [reply apps/builder/app/builder/features/workspace/canvas-tools/canvas-tools.tsx:71]: true

**Pull request merged**